install.sh

```diff
@@ -280,9 +280,6 @@ echo_out "Set up and synchronize local package repo..."
 echo "LD_LIBRARY_PATH=$CREW_PREFIX/lib${LIB_SUFFIX}:/lib${LIB_SUFFIX}" >> "$CREW_PREFIX"/etc/env.d/00-library
 export LD_LIBRARY_PATH="${CREW_PREFIX}/lib${LIB_SUFFIX}:/lib${LIB_SUFFIX}"
 
-# Add the CREW_PREFIX bin and musl bin directories to PATH.
-echo -e "## Inserted by Chromebrew's install.sh\nPATH=$CREW_PREFIX/bin:$CREW_PREFIX/sbin:$CREW_PREFIX/share/musl/bin:\$PATH" > "$CREW_PREFIX"/etc/env.d/path
-
 echo "export CREW_PREFIX=${CREW_PREFIX}" >> "${CREW_PREFIX}/etc/env.d/profile"
```

Comment:
Should this be in a package instead?  Otherwise, there is no way to uninstall on crew removal.

Reply:
it's in crew_profile_base...